docs/application/dotnet/guides/nui/flex-layout.md

```diff
@@ -110,8 +110,80 @@ flexLayout.WrapType = FlexLayout.FlexWrapType.NoWrap;
 layoutView.Layout = flexLayout;
 ```
 
+The following properties affect the specified child view areas:
+- `FlexAlignmentSelf`
+- `FlexPositionType`
+- `FlexAspectRatio`
+- `FlexBasis`
+- `FlexShrink`
+- `FlexGrow`
 
-## Related Information
+> [!NOTE]
+> The `FlexAlignmentSelf`, `FlexPositionType`, `FlexAspectRatio`, `FlexBasis`, `FlexShrink`, and `FlexGrow` properties are added in Tizen 6.0.
+
+**Table: Properties and descriptions**
+
+| Property               | Type            | Description  |
+| -----------------------| --------------- | ------------ |
+| `FlexAlignmentSelf`    | AlignmentType   | The alignment of the item along the cross axis when free space is available. |
+| `FlexPositionType`     | PositionType    | The position type defines how the item is positioned within its parent. |
+| `FlexAspectRatio`      | float           | The ratio between the width and the height of an item. |
+| `FlexBasis`            | float           | Specifies item's default size before `FlexGrow` and `FlexShrink` calculations are performed. |
+| `FlexShrink`           | float           | Specifies how to shrink children along the main axis in case the total size of the children overflows the size of the flex layout on the main axis. |
+| `FlexGrow`             | float           | Specifies how the space within a flex layout is distributed among its children along the main axis. |
+
+### FlexAlignmentSelf property
```

Comment:
After the second level heading in line 93 ## FlexWrapType, the next heading is level 3
The staging link is not working, so I am unable to confirm how the right-side TOC will appear.

Is the intention that all the third-level headings are to be grouped under FlexWrapType? : 
 ## FlexWrapType
### FlexAlignmentSelf property
### FlexPositionType property
### FlexAspectRatio property
### FlexBasis property
### FlexShrink property
### FlexGrow property

If not can we have a meaningful second-level heading under which these third-level headings can be grouped?

Reply:
second-level heading `Attached Property` is added.

Reply:
@dags34, thanks for fixing.